### PR TITLE
EPMRPP-76310 || Add animations to landing pricing page

### PR DESCRIPTION
**Pull request by Vadim73i**

### CODE REVIEW

File: src/react-components/common/info-with-tooltip/infoWithTooltip.scss [hunk 1]
```diff
@@ -24,15 +24,18 @@
...
-  display: none;
-  transition-delay: 500ms;
-  transition-property: display;
   z-index: 1;
   position: fixed;
   width: 207px;
+  display: block;
+  opacity: 0;
+  visibility: hidden;
+  transition: opacity 0.3s, visibility 0.3s;
```
Comment (AmsterGet) [inline src/react-components/common/info-with-tooltip/infoWithTooltip.scss:33]: `visibility` property is not animated.
See https://stackoverflow.com/questions/27900053/css-transition-with-visibility-not-working

File: src/react-components/common/info-with-tooltip/infoWithTooltip.scss [hunk 2]
```diff
@@ -24,15 +24,18 @@
...
 
   &.visible {
-    display: block;
+    z-index: 1;
```
Comment (AmsterGet) [inline src/react-components/common/info-with-tooltip/infoWithTooltip.scss]: Please move it to SCSS variables.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: src/common/css/variables/color-variables.scss [hunk 3]
```diff
@@ -25,6 +25,7 @@ $COLOR--dark-pastel-green: #5CB755;
 $COLOR--skipped: #bdc7cc;
 $COLOR--error-red: #EC3900;
 $COLOR-WHITE-CARD-SHADOW: #eaeced;
+$COLOR-WHITE-CARD-SHADOW-WITH-OPACITY: #eaeced90;
```
Comment (chivekrodis) [inline src/common/css/variables/color-variables.scss]: Does it really necessary extract into own property color with opacity?
maybe better do it directly in the file like `rgba(color, opacity)`

File: src/react-components/common/custom-checkbox/customCheckbox.scss [hunk 4]
```diff
@@ -26,6 +26,7 @@
     background: $COLOR-WHITE_TWO;
     border-radius: 3px;
     border: solid 2px $COLOR-TEXT-INFO;
+    transition: 0.3s;
```
Comment (chivekrodis) [inline src/react-components/common/custom-checkbox/customCheckbox.scss]: better add property which you want to animate. Animate everything is not such good practice

File: src/react-components/plan-card/simple-card/simpleCard.scss [hunk 5]
```diff
@@ -14,24 +14,66 @@
...
+      margin-bottom: calc(-#{$cardScale} * 2);
+    }
+  }
+
+  &:hover {
+    height: calc(#{$height} + #{$cardScale} * 2);
+    width: calc(#{$width} + #{$cardScale} * 2);
+    bottom: calc(#{$cardBottom} + #{$cardScale});
+    box-shadow: 0 4px 15px $COLOR-WHITE-CARD-SHADOW-WITH-OPACITY;
+
```
Comment (chivekrodis) [inline src/react-components/plan-card/simple-card/simpleCard.scss]: I guess you can create the same scale animation with `transform: scale(1.05)` property without complicate the code

**Review by chivekrodis — CHANGES_REQUESTED** [review]

**Review by chivekrodis — APPROVED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**